src/root_files/app/front-channel.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Deriv</title>
+    <meta charset="utf-8" />
+    <script>
+      try {
+        // we check if we can access the top.location, if we can't it will throw an error
+        // and this means this front-channel is not rendered on the same window as the application which is logging out
+        let isSameOrigin = top.location.hostname === self.location.hostname;
+      } catch (err) {
+        localStorage.removeItem("client.accounts");
+        sessionStorage.removeItem("active_loginid");
+        localStorage.removeItem("active_loginid");
+
+        // clear elev.io session storage
+        sessionStorage.removeItem("_elevaddon-6app");
+        sessionStorage.removeItem("_elevaddon-6create");
+        // clear trading session
+        localStorage.removeItem("ws_cache");
+      }
```

Comment:
This will be called by OIDC in an iframe when you log out in other applications like Deriv.app/DBot/P2P, thus it will clear the local storage in SmartTrader and log the user out